src/main/com/smithsmodding/armory/common/crafting/blacksmiths/recipe/VanillaAnvilRecipe.java

```diff
@@ -183,7 +172,8 @@ private void calculateValues() {
             if (iRightInputStack != null) {
                 if (!checkForge(iLeftInputStack, iRightInputStack, iEntity.getState().getItemName(), tBaseLevelExperienceCost))
                     return;
-                tFlagIsEnchantableByBook = iRightInputStack.getItem() == Items.ENCHANTED_BOOK && Items.ENCHANTED_BOOK.getEnchantments(iRightInputStack).tagCount() > 0;
+
+                tFlagIsEnchantableByBook = iRightInputStack.getItem() == Items.ENCHANTED_BOOK && ItemEnchantedBook.getEnchantments(iRightInputStack).tagCount() > 0;
 
                 if (tEventStack.isItemStackDamageable() && tEventStack.getItem().getIsRepairable(iLeftInputStack, iRightInputStack)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)